phpBB/includes/acp/acp_search.php

```diff
@@ -409,15 +416,36 @@ private function index_action(string $id, string $mode, string $action): void
 				$u_action = append_sid($this->phpbb_admin_path . "index." . $this->php_ex, "i=$id&mode=$mode&action=$action&hash=" . generate_link_hash('acp_search'), false);
 				meta_refresh(1, $u_action);
 
-				$message_redirect = $this->language->lang(($action == 'create') ? 'SEARCH_INDEX_CREATE_REDIRECT' : 'SEARCH_INDEX_DELETE_REDIRECT', (int) $status['row_count'], $status['post_counter']);
-				$message_rate = $this->language->lang(($action == 'create') ? 'SEARCH_INDEX_CREATE_REDIRECT_RATE' : 'SEARCH_INDEX_DELETE_REDIRECT_RATE', $status['rows_per_second']);
-				trigger_error($message_redirect . $message_rate);
+				$message_progress = $this->language->lang(($action === 'create') ? 'INDEXING_IN_PROGRESS' : 'DELETING_INDEX_IN_PROGRESS');
+				$message_progress_explain = $this->language->lang(($action == 'create') ? 'INDEXING_IN_PROGRESS_EXPLAIN' : 'DELETING_INDEX_IN_PROGRESS_EXPLAIN');
+				$message_redirect = $this->language->lang(
+					($action === 'create') ? 'SEARCH_INDEX_CREATE_REDIRECT' : 'SEARCH_INDEX_DELETE_REDIRECT',
+					(int) $status['row_count'],
+					$status['post_counter']
+				);
+				$message_redirect_rate = $this->language->lang(
+					($action === 'create') ? 'SEARCH_INDEX_CREATE_REDIRECT_RATE' : 'SEARCH_INDEX_DELETE_REDIRECT_RATE',
+					$status['rows_per_second']
+				);
+
+				$this->template->assign_vars([
+					'INDEXING_TITLE'		=> $this->language->lang($message_progress),
+					'INDEXING_EXPLAIN'		=> $this->language->lang($message_progress_explain),
```

Comment:
If my vision is not playing tricks on me, then here we see re-assigning language vars.. Previously assigned on lines 419-420.

Reply:
You're right, resolved now.